Sources/NIOHTTP1Server/main.swift

```diff
@@ -147,7 +147,7 @@ private final class HTTPHandler: ChannelInboundHandler {
                 self.buffer.write(string: "line \(self.continuousCount)\n")
                 ctx.writeAndFlush(self.wrapOutboundOut(.body(.byteBuffer(self.buffer)))).map {
                     _ = ctx.eventLoop.scheduleTask(in: .milliseconds(400), doNext)
-                }.whenFailure { error in
+                }.whenFailure { _ in
```

Comment:
Our preferred policy when we use the `_` is to provide the type annotation (e.g. `_: Error`). This helps the compiler spot (and give useful feedback) if type signatures change underneath you. 

Mind updating to do that?